user_tools/src/spark_rapids_pytools/cloud_api/dataproc.py

```diff
@@ -429,8 +448,8 @@ def parse_accelerator_description(raw_description: str) -> dict:
                              gpu_mem=gpu_configs.get('gpu_mem'))
 
     def _pull_sys_info(self, cli=None) -> SysInfo:
-        cpu_mem = self.mc_props.get_value('memoryMb')
-        num_cpus = self.mc_props.get_value('guestCpus')
+        cpu_mem = self.mc_props.get_value('MemoryInMB')
```

Comment:
Similar to the previous comment. It seems same as the implementation for `emr` and `databricks-azure`. We could probably move this to base class.

Reply:
I will note this and clean this up in the final PR (for final refactoring and removing dead code) if that sounds okay.